tests/unit/forklift/test_legacy.py

```diff
@@ -2593,8 +2594,17 @@ def test_upload_succeeds_with_2fa_enabled(
             IMetricsService: metrics,
         }.get(svc)
 
-        legacy.file_upload(db_request)
+        with pytest.raises(BasicAuthTwoFactorEnabled) as excinfo:
```

Comment:
probably change name of this test now

Reply:
Eh? Is the change to `test_basic_auth_upload_fails_with_2fa_enabled` not sufficient?

Reply:
![image](https://github.com/pypi/warehouse/assets/1200832/a64fd868-fb87-4461-bebb-6420225d20d8)

I guess weird folding in UI